app/project_type/directory.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, project_directory, config=None, job_name=None, build_project_
         self.project_directory = os.path.abspath(project_directory)
         self._logger.debug('Project directory is {}'.format(project_directory))
 
-    def _setup_build(self):
+    def _fetch_project(self):
         check_command = 'test -d "{}"'.format(self.project_directory)
         output, exit_code = self.execute_command_in_project(check_command, cwd='/')
         if exit_code != 0:
```

Comment:
We had originally named it "setup_build" because it was the do-everything method, and it made sense for it to be broad. I knew for sure that we wanted to rename it from setup_build() now that the function doesn't do everything anymore, but I'm pretty open to renaming it; though I do like "fetch_project".